BlazorAccessibility/Shared/AccordionPanel.razor

```diff
@@ -26,10 +26,11 @@
 
 
 @code {
+    private bool isHidden => !IsExpanded;
+    private bool preventKey;
+    private string Role => IsExpanded ? "region" : null!;
+
     public bool IsExpanded { get; set; } = false;
-    bool isHidden => !IsExpanded;
-    public string role => IsExpanded ? "region" : null!;
-    bool preventKey;
```

Comment:
Changed role to be private. If you make `IPanel` internal, then you can also make `IsExpanded` internal, to prevent users from changing the value from the outside.